helmfile/charts/networkpolicy/common/templates/trivy/allow-trivy-vulnerability-scanner.yaml

```diff
@@ -0,0 +1,25 @@
+{{ if .Values.monitoring.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-trivy-vulnerability-scanner
+  namespace: monitoring
+spec:
+  podSelector:
+    matchLabels:
+      vulnerabilityReport.scanner: Trivy
+  policyTypes:
+    - Egress
+  egress:
```

Comment:
Does it not get scraped by prometheus anymore?

Reply:
Yes it does, via the operator.